### PR TITLE
Feature/at 5886 User list view pagination

### PR DESCRIPTION
**Pull request by tonysumner-ccpo**
[AT-5886 User list view pagination](https://ccpo.atlassian.net/browse/AT-5886?atlOrigin=eyJpIjoiYWM1OWI0YjFhZTdmNGZjYWE3YmFlYjViOTE2YTU0MTAiLCJwIjoiaiJ9)

[AT-5887 User search by keyword](https://ccpo.atlassian.net/browse/AT-5887?atlOrigin=eyJpIjoiZGMxNDNjYTEyMDdlNDY0MWJkYzc4ZWEwYTlmM2Q3MDAiLCJwIjoiaiJ9)

[AT-5888 User filter by organization](https://ccpo.atlassian.net/browse/AT-5888?atlOrigin=eyJpIjoiNmMwMDUwM2IyOGE1NGU0OGI0ZGE1MzhjODBiODJlYTIiLCJwIjoiaiJ9)

**AT-5886 User list view pagination**
As a CCPO admin, I want the User List View to be paginated, so that I don’t have an endless scroll that takes a long time to load on the page.

This will be functionally equivalent to: AT-5851: Portfolio List View paginationTO DO 

See below for an image of the component, and see design specs on the Users List View page on Abstract. 

Note: Abstract/Invision links will only be available until June 2021. A PNG version of the view from the above Abstract link is retained for reference on Confluence here: Users List View.png. Please see Design Files and Assets  in Confluence for instructions on how you can view design specs using Sketch.

**AT-5887 User search by keyword**
As a CCPO admin, I want to perform a keyword search, so that I can find a user based on what I know about them.

This is functionally similar to 

Given I enter characters into the search bar, when I click the search button, then I should see a list of users that contain my searched term in any of the below categories as shown by the view columns:

- Name
- Organization
- DoD ID
- Portfolio(s)

See below for the component, and see design specs on the Abstract Users List View page. 

Note: Abstract/Invision links will only be available until June 2021. A PNG version of the view from the above Abstract link is retained for reference on Confluence here: Users List View.png. Please see   in Confluence for instructions on how you can view design specs using Sketch.

Additionally if no results are found, then the following should be displayed to the user:

**AT-5888 User filter by organization**

As a CCPO admin, I want to filter by organization, so that I can find all users associated with a specific DoD organization.

This is functionally similar to 

Given I click the dropdown arrow to filter by organization, when I select an organization, then I should see all users associated with that organization.

Dropdown options include:

- Air Force
- Army
- Combatant Command
- Defense Agency and Field Activity (DAFA)
- Joint Staff
- Marine Corps
- Navy
- Office of the Secretary of Defense (OSD) / Principal Staff Assistants (PSAs)
- Space Force

See below for an image of the component and see design specs in the Abstract Users List View page.

Note: Abstract/Invision links will only be available until June 2021. A PNG version of the view from the above Abstract link is retained for reference on Confluence here: Users List View.png. Please see   in Confluence for instructions on how you can view design specs using Sketch.

Additionally if there is no user in the selected organization, then the results should look like the following image.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -9,25 +9,32 @@
     "build-prod": "parcel build js/index.js -d static/assets --public-url $CDN_URL -o index.js",
     "test": "jest",
     "test:coverage": "jest --coverage --collectCoverageFrom='js/**/*.js'",
+    "test:VueJScoverage": "jest --coverage --collectCoverageFrom='js/**/*.vue'",
```
Comment (tonysumner-ccpo) [inline package.json:12]: to test Jest tests for a fully structured Vue component.  I added the first fully structured Vue component this week.

**Review by jGonzalez-CCPO — CHANGES_REQUESTED** [review]
how we can get this better?

3 Code Smells
86.0% 86.0% Coverage

**Review by jGonzalez-CCPO — CHANGES_REQUESTED** [review]
there is still coverage and smells pending:

Code Smell A 10 Code Smells
89.8% 89.8% Coverage

**Comment by jGonzalez-CCPO** [comment]
love this stats: 

Bug A 0 Bugs
Vulnerability A 0 Vulnerabilities
Security Hotspot A 0 Security Hotspots
Code Smell A 0 Code Smells

94.0% 94.0% Coverage
0.0% 0.0% Duplication

**Review by jGonzalez-CCPO — CHANGES_REQUESTED** [review]

**Review by jGonzalez-CCPO — APPROVED** [review]

File: atat/routes/admin/admin.py [hunk 2]
```diff
@@ -99,6 +99,11 @@ def all_users_page():
         order_asc = "1"
 
     users_info_json = json.dumps(users_info)
+    service_branches_description = []
+    for branch in SERVICE_BRANCHES:
+        service_branches_description.append(branch[1])
+
+    service_branches_description_json = json.dumps(service_branches_description)
```
Comment (jGonzalez-CCPO) [inline atat/routes/admin/admin.py:106]: if you want the list to be sorted: 

service_branches_description_json = json.dumps(service_branches_description).sort())

Comment (jGonzalez-CCPO) [reply atat/routes/admin/admin.py:106]: but I think the order is the exact order requested by the DOD that is a custom order not alphabetic so Other is the last and the branches are in order of use.

**Pull request merged**